Importer fails to download files from minio in local node

### Description

Importer fails to download files from minio s3 service in local node:

```
2022-11-17T08:13:34.850-0600 INFO scheduling-2 c.h.m.i.d.r.RecordFileDownloader No new signature files to download after file: 2022-10-25T17_57_27.968419Z.rcd. Retrying in 0.5 s
2022-11-17T08:13:35.565-0600 WARN sdk-async-response-1-7 c.h.m.i.d.p.CompositeStreamFileProvider Attempt #1 failed: Received an UnknownHostException when attempting to interact with a service. See cause for the exact endpoint that is failing to resolve. If this is happening on an endpoint that previously worked, there may be a network connectivity issue or your DNS cache could be storing endpoints for too long.
2022-11-17T08:13:35.565-0600 ERROR pool-13-thread-29 c.h.m.i.d.r.RecordFileDownloader Error downloading signature files for node 0 after 213.2 ms software.amazon.awssdk.core.exception.SdkClientException: Received an UnknownHostException when attempting to interact with a service. See cause for the exact endpoint that is failing to resolve. If this is happening on an endpoint that previously worked, there may be a network connectivity issue or your DNS cache could be storing endpoints for too long.\
	at software.amazon.awssdk.core.exception.SdkClientException$BuilderImpl.build(SdkClientException.java:102)\
	at software.amazon.awssdk.awscore.interceptor.HelpfulUnknownHostExceptionInterceptor.modifyException(HelpfulUnknownHostExceptionInterceptor.java:59)\
	at software.amazon.awssdk.core.interceptor.ExecutionInterceptorChain.modifyException(ExecutionInterceptorChain.java:202)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.utils.ExceptionReportingUtils.runModifyException(ExceptionReportingUtils.java:54)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.utils.ExceptionReportingUtils.reportFailureToInterceptors(ExceptionReportingUtils.java:38)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.AsyncExecutionFailureExceptionReportingStage.lambda$execute$0(AsyncExecutionFailureExceptionReportingStage.java:49)\
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)\
	at software.amazon.awssdk.utils.CompletableFutureUtils.lambda$forwardExceptionTo$0(CompletableFutureUtils.java:76)\
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)\
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)\
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.AsyncRetryableStage$RetryingExecutor.maybeAttemptExecute(AsyncRetryableStage.java:103)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.AsyncRetryableStage$RetryingExecutor.maybeRetryExecute(AsyncRetryableStage.java:185)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.AsyncRetryableStage$RetryingExecutor.lambda$attemptExecute$1(AsyncRetryableStage.java:160)\
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)\
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)\
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)\
	at software.amazon.awssdk.utils.CompletableFutureUtils.lambda$forwardExceptionTo$0(CompletableFutureUtils.java:76)\
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)\
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)\
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.MakeAsyncHttpRequestStage.lambda$null$0(MakeAsyncHttpRequestStage.java:103)\
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)\
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)\
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.MakeAsyncHttpRequestStage.lambda$executeHttpRequest$3(MakeAsyncHttpRequestStage.java:165)\
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)\
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)\
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)\
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\
	at java.base/java.lang.Thread.run(Thread.java:833)\
	Suppressed: java.lang.Exception: #block terminated with an error\
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:99)\
		at reactor.core.publisher.Mono.block(Mono.java:1707)\
		at com.hedera.mirror.importer.downloader.Downloader.lambda$downloadAndParseSigFiles$4(Downloader.java:248)\
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)\
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\
		... 3 more\
Caused by: software.amazon.awssdk.core.exception.SdkClientException: Unable to execute HTTP request: hedera-streams.minio\
	at software.amazon.awssdk.core.exception.SdkClientException$BuilderImpl.build(SdkClientException.java:102)\
	at software.amazon.awssdk.core.exception.SdkClientException.create(SdkClientException.java:47)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.utils.RetryableStageHelper.setLastException(RetryableStageHelper.java:211)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.utils.RetryableStageHelper.setLastException(RetryableStageHelper.java:207)\
	at software.amazon.awssdk.core.internal.http.pipeline.stages.AsyncRetryableStage$RetryingExecutor.maybeRetryExecute(AsyncRetryableStage.java:183)\
	... 22 more\
Caused by: java.net.UnknownHostException: hedera-streams.minio\
```

In release 0.68.0, mirrornode bumped up AWS SDK which changed the default behavior of the s3 client to prepend the bucket name to the hostname, this causes the  `UnknownHostException` in local node env.

### Steps to reproduce

Run local node with mirrornode release v0.68.0

### Additional context

_No response_

### Hedera network

other

### Version

v0.68.0

### Operating system

_No response_

--- a/hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/config/CloudStorageConfiguration.java
+++ b/hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/config/CloudStorageConfiguration.java
@@ -104,6 +104,7 @@ class CloudStorageConfiguration {
         return S3AsyncClient.builder()
                 .credentialsProvider(awsCredentialsProvider(sourceProperties))
                 .endpointOverride(sourceProperties.getUri())
+                .forcePathStyle(true)
                 .httpClient(httpClient)
                 .overrideConfiguration(c -> c.addExecutionInterceptor(metricsExecutionInterceptor))
                 .overrideConfiguration(overrideProject(sourceProperties))

